operator/kafkacontroller/observe_test.go

```diff
@@ -351,7 +351,7 @@ func TestObserve_Outdated_Settings(t *testing.T) {
 		assert.NoError(t, err)
 		require.NotNil(t, res)
 		assert.True(t, res.ResourceExists, "report resource exits")
-		assert.False(t, res.ResourceUpToDate, "report resource not uptodate")
+		assert.False(t, res.ResourceUpToDate, "report resource not updated")
```

Comment:
I wouldn't this change, the resource is actually "not up to date", right?

Reply:
ah yes, I probably miss read it.